### ISSUE TITLE
Packrat: .RPROFILE Improval

### ISSUE THREAD

**Issue by jtelleria** [rstudio/packrat#485]
A Known limitation of packrat are that the project Level .RProfile overrides the System default .RProfile

And this can be easily fixed with a single line of code in the .RProfile packrat creates by default:

```
if(file.exists("~/.Rprofile")) source("~/.Rprofile")
```

Therefore, I could do a "Pull Request" with this line fix to the packrat source code if appropriate.

Thank you.
Juan

**Comment by kevinushey** [comment]
I don't think we want Packrat to source a global .Rprofile by default -- the intention is for Packrat projects to be encapsulated from the 'global' R environment, and this could potentially break that encapsulation.

That said, I think this is not unreasonable -- it would just need to be activated in a project-to-project basis, I think. The associated code could be added here:

https://github.com/rstudio/packrat/blob/7d320b1c946a5c71232baeba8bc1fa96e9e6befb/inst/resources/init.R#L21-L28

and would likely need to be something of the form:

```R
.Rprofile <- getOption("packrat.global.profile", unset = "")
if (file.exists(.Rprofile))
  source(.Rprofile)
```

where `packrat.global.profile` could be set by the user to e.g. `~/.Rprofile` as needed.

**Comment by kevinushey** [comment]
Actually, a better solution would be to make use of a project-local `.Renviron` file, since this gets read before any startup profiles. So a file `.Renviron` could be added with the contents:

```R
R_PACKRAT_GLOBAL_PROFILE=~/.Rprofile
```

and then it could be used with e.g.

```R
.Rprofile <- Sys.getenv("R_PACKRAT_GLOBAL_PROFILE")
if (file.exists(.Rprofile))
  source(.Rprofile)
```
